crates/cli/src/main.rs

```diff
@@ -13,28 +13,28 @@ use std::{
 use util::paths::PathLikeWithPosition;
 
 #[derive(Parser, Debug)]
-#[clap(name = "zed", global_setting(clap::AppSettings::NoAutoVersion))]
+#[command(name = "zed")]
 struct Args {
     /// Wait for all of the given paths to be opened/closed before exiting.
-    #[clap(short, long)]
+    #[arg(short, long)]
     wait: bool,
     /// Add files to the currently open workspace
-    #[clap(short, long, overrides_with = "new")]
+    #[arg(short, long, overrides_with = "new")]
     add: bool,
     /// Create a new workspace
-    #[clap(short, long, overrides_with = "add")]
+    #[arg(short, long, overrides_with = "add")]
     new: bool,
     /// A sequence of space-separated paths that you want to open.
     ///
     /// Use `path:line:row` syntax to open a file at a specific location.
     /// Non-existing paths and directories will ignore `:line:row` suffix.
-    #[clap(value_parser = parse_path_with_position)]
+    #[arg(value_parser = parse_path_with_position)]
     paths_with_position: Vec<PathLikeWithPosition<PathBuf>>,
     /// Print Zed's version and the app path.
-    #[clap(short, long)]
+    #[arg(short = 'V', long)]
```

Comment:
```suggestion
#[arg(short, long)]
```

Reply:
This would be a breaking change, as the short version for the "version" command line argument in clap is an uppercase "V". Is this breaking change desirable?

Reply:
I think that this wouldn't be a breaking change, as we're explicitly disabling clap's `--version` with `disable_version_flag`/`clap::AppSettings::NoAutoVersiob`

Reply:
I've decided to check. With `NoAutoVersion`, it is indeed a lowercase "v".